### DISCUSSION TITLE
Add !important utility

### DISCUSSION THREAD

**Discussion by phmasek** [tailwindlabs/tailwindcss#3033] [Ideas]
Would be really helpful to have the ability to add `!important` just in the same way as the [responsive designs](https://tailwindcss.com/docs/responsive-design) and [states](https://tailwindcss.com/docs/hover-focus-and-other-states) are handled.

An idea is to have the prefix pseudo element `imp:`. 
For example: `imp:text-white`

**Comment by phmasek** [comment]
Yeah... never mind me. I found it in [the docs](https://tailwindcss.com/docs/plugins#complex-variants).

**Reply by fosterushka** [reply]
Thanks man, i am like looking for this about 10 times at docs and didn't see that
